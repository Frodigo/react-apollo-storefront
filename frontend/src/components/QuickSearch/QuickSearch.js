import React from 'react';
import { Form, FormControl, Button } from 'react-bootstrap';

const QuickSearch = () => {
    return (
        <Form inline className="w-100 justify-content-center">
            <FormControl type="text" placeholder="Search entire shop" className="mr-2 w-75"/>
            <Button variant="outline-success">
                <i className="bi bi-search"></i>
            </Button>
        </Form>
    );
};


export default QuickSearch;
