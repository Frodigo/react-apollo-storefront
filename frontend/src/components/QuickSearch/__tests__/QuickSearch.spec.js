import React from 'react';
import { render } from '@testing-library/react';

import QuickSearch from '../QuickSearch';

test('QuickSearch renders correctly', () => {

    const { container } = render(<QuickSearch />);

    expect(container).toMatchSnapshot();
});

