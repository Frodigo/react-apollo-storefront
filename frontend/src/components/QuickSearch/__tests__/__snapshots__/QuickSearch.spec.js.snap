// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuickSearch renders correctly 1`] = `
<div>
  <form
    class="w-100 justify-content-center form-inline"
  >
    <input
      class="mr-2 w-75 form-control"
      placeholder="Search entire shop"
      type="text"
    />
    <button
      class="btn btn-outline-success"
      type="button"
    >
      <i
        class="bi bi-search"
      />
    </button>
  </form>
</div>
`;
